css/style.css

```diff
@@ -0,0 +1,506 @@
+}
+
+.header_logo-icon {
+  display: flex;
+}
+
+.logo-icon__item {
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: 0.7;
+}
+
+.logo-icon__item-blue {
+  position: relative;
+  z-index: 1;
+  background-color: #0056ec;
+  left: 23px;
+}
+
+.logo-icon__item-white {
+  position: relative;
+  z-index: 2;
+  background-color: #c8dbfb;
+  top: 8px;
+  left: 4px;
+}
+
+.header__logo {
+  font-family: 'PT Serif', serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  padding: 0 0 0 12px;
+}
+
+.header__menu {
+  display: flex;
+  width: 383px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.menu__item {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 21px;
+  color: #000;
+  display: inline-block;
+}
+
+.menu__item::after {
+  display: block;
+  content: "";
+  height: 3px;
+  width: 0%;
+  background-color: #0056EC;
+  transition: .4s ease-in-out;
+}
+
+.menu__item:hover::after,
+.menu__item:focus::after {
+  width: 100%;
+}
+
+.menu__item_active::after {
+  display: block;
+  content: "";
+  height: 3px;
+  width: 100%;
+  background-color: #0056EC;
+}
+
+/* content */
+.content {
+  display: flex;
+  max-width: 1518px;
+  height: 659px;
+  margin: 0 auto;
+  margin-bottom: 131px;
+  flex-direction: row;
+}
+
+.content__wrap {
+  height: 445px;
+  max-width: 567px;
+  align-items: center;
+  margin-right: 203px;
+  padding: 27px 0 0 0;
+}
+
+.content__title {
+  max-width: 567px;
+  height: 255px;
+  font-family: 'PT Serif', serif;
+  font-weight: bold;
+  font-size: 66px;
+  line-height: 87px;
+}
+
+.content__title > .content__title_gray {
+  background-color: #ecf3ff;
+  padding: 2px 0 2px 0;
+  line-height: 1.6;
+}
+
+.content__desc {
+  max-width: 455px;
+  height: 52px;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 26px;
+  margin-bottom: 40px;
+}
+
+.content__bt {
+  max-width: 270px;
+  height: 57px;
+  background-color: #0056EC;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+}
+
+.content__bt__link {
+  text-decoration: none;
+  color: #fff;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  margin: 0 auto;
+}
+
+.content__img {
+  background-image: url(../img/big.png);
+  background-repeat: no-repeat;
+  height: 657px;
+  width: 770px;
+}
+
+/* footer */
+.footer {
+  width: 1888px;
+  margin: 0 auto;
+}
+
+.footer__scroll {
+  width: 32px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+}
+
+.scroll-item {
+  background-color: #C8DBFB;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+}
+
+@media (max-width: 1450px) {
+  .content__title {
+    font-size: 60px;
+  }
+}
+
+@media (max-width: 1366px) {
+  .wrap__site {
+    max-width: 1366px;
+  }
+
+  .header {
+    max-width: 960px;
+    margin: 35px auto 76px;
+  }
+
+  .header__menu {
+    max-width: 336px;
+  }
+
+  .menu__item {
+    font-size: 14px;
+    line-height: 16px;
+    padding: 7px 16px 0 0;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)